### ISSUE TITLE
FAILED TO LOAD THE RENDERDOC MODULE

### ISSUE THREAD

**Issue by aladebaran** [eliemichel/MapsModelsImporter#174]
Information provided:

 - I don't know how to post the .rdc file. How can I? 

 - My GPU is Nvidia gtx  660 
 - Add-on version is 0.3.7
 - RenderDoc version is 1.10 
 - My Blender version is 2.93

**Comment by mika314** [comment]
I have a similar issue:

```
Read prefs: D:\Users\teant\AppData\Roaming\Blender Foundation\Blender\2.93\config\userpref.blend

==========================================================================================
google_maps_rd failed and returned:
Error: Failed to load the RenderDoc Module. It however seems to exist.
This might be due to one of the following reasons:
 - Your Blender version uses another version of python than used to build the RenderDoc Module
 - An additional file required by the RenderDoc Module is missing (i.E. renderdoc.dll)
 - Something completely different

Remember, you must use exactly the same version of python to load the RenderDoc Module as was used to build it.
Find more information about building the RenderDoc Module here: https://github.com/baldurk/renderdoc/blob/v1.x/docs/CONTRIBUTING/Compiling.md

sys.platform:  win32
Python version:  3.9.2 (default, Mar  1 2021, 08:18:55) [MSC v.1916 64 bit (AMD64)]
err.name:  renderdoc
err.path:  D:\Users\teant\AppData\Roaming\Blender Foundation\Blender\2.93\scripts\addons\MapsModelsImporter\bin\win64\renderdoc.pyd
Error Message:  DLL load failed while importing renderdoc: The specified module could not be found.



Extra info:
 - python = D:\Program Files (x86)\Steam\steamapps\common\Blender\2.93\python\bin\python.EXE
 - python_home = D:\Program Files (x86)\Steam\steamapps\common\Blender\2.93\python

```

RenderDoc: 1.10
MapsModelsImporter: 0.3.7

Blender: 2.93

**Comment by mika314** [comment]
I made it work with 2.92. I am not sure what helped tho, a downgrade of the fact I installed Blender from the website, not from Steam.

**Comment by eliemichel** [comment]
Duplicate of #172, now fixed with v0.4.0
